src/coreclr/jit/emit.cpp

```diff
@@ -4657,7 +4657,7 @@ bool emitter::emitEndsWithAlignInstr()
 //
 //  Returns:  size of a loop in bytes.
 //
-unsigned emitter::getLoopSize(insGroup* igLoopHeader, unsigned maxLoopSize)
+unsigned emitter::getLoopSize(insGroup* igLoopHeader, unsigned maxLoopSize DEBUG_ARG(bool isAlignAdjusted))
```

Comment:
The new arg needs to be documented above.

Reply:
done.